src/Components/Publishing/Display/Canvas/index.tsx

```diff
@@ -5,19 +5,21 @@ import React from "react"
 import track from "react-tracking"
 import Waypoint from "react-waypoint"
 import styled, { StyledFunction } from "styled-components"
+import { isAdmin } from "Utils/admin"
 import { ErrorBoundary } from "../../../ErrorBoundary"
 import { pMedia } from "../../../Helpers"
 import { getCurrentUnixTimestamp } from "../../Constants"
 import { PixelTracker, replaceWithCacheBuster } from "../ExternalTrackers"
 import { trackImpression, trackViewability } from "../track-once"
 import { CanvasContainer, unitLayout } from "./CanvasContainer"
-
 interface DisplayCanvasProps {
   unit: any
   campaign: any
   article?: any
   renderTime?: number
   tracking?: any
+  adUnit: string
```

Comment:
I would type `adUnit` and `adDimension` using the enums declared below since the values are so restricted values, that way Typescript can help us catch errors.